image-types/rhel8/amazon-ec2.md

```diff
@@ -0,0 +1,79 @@
+# Amazon EC2 Image
+
+This image is meant to be used in [Amazon Elastic Compute Cloud (EC2)][ec2], a
+popular cloud computing platform. It has to follow Amazon’s
+[guidelines][guidelines] and [requirements][requirements] for shared AMIs.
+
+## File Format
+
+EC2 uses Amazon Machine Images (AMIs) internally, which only be created inside
+EC2. An image in a standard format (ova, vmdk, vhd/x, or raw) must be imported
+from S3 storage. **osbuild-composer** generates this image type in the VHDX
+format, because it is fairly modern, widely used, and uses less space than RAW
+images.
+
+The default image size is 6 GB, unless it has to be larger due to installed
+packages.
+
+## Architectures
+
+Only `x86_64` images are generated.
```

Comment:
I've verified that lorax-composer can also build arch64 AMIs and we can boot them in AWS. Brian made some patches in this area.

Do you mean that only x86_64 is supported b/c osbuild-composer is lacking multi-arch support or simply b/c this was not tested ?

Reply:
Oops, osbuild-composer actually creates `aarch64` images as well. Thanks!